### PR TITLE
feat(info): add sharing call-to-action

### PR DESCRIPTION
**Pull request by alexissmirnov**
please see the change to `package-lock.json` -- I didn't do it myself, but a fresh `npm install` did it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ yarn-debug.log*
 yarn-error.log*
 
 src/content/manifest.json
+package-lock.json
```
Comment (vicrep) [inline .gitignore]: Should _not_ be gitignored

**Comment by vicrep** [comment]
Please remove package-lock changes from this PR :)

The reason there were changes is that you're probably running an old version of NPM, but given that you're not intending on actually updating anything, easiest resolution is just to remove / not add the changes your NPM does to package-lock

File: src/components/share-results.tsx [hunk 2]
```diff
@@ -59,14 +66,18 @@ export const ShareResults: React.FC<Props> = (props: any) => {
...
-      <ShareContainer>
-        <FacebookShareButton href={facebookHref}>
-          {t('share.facebookButton')}
-        </FacebookShareButton>
-        <TwitterSharebutton href={twitterHref}>
-          {t('share.twitterButton')}
-        </TwitterSharebutton>
-      </ShareContainer>
+      <div>
+        <CtaText>{t('share.CTA')}</CtaText>
```
Comment (vicrep) [inline src/components/share-results.tsx]: Looks odd that it's wrapped in another div, it should be within the `ShareContainer` imo if possible

**Comment by vicrep** [comment]
@alexissmirnov actually, the change to package-lock makes sense, it was my fault for not updating it when I renamed the npm project. So changes are good, please just remove package-lock.json from your gitignore

File: src/components/share-results.tsx (hunk 2 above)
Comment (alexissmirnov) [reply src/components/share-results.tsx]: putting it in a ShareContainer will make the text be on the left side of the sharing buttons

Comment (vicrep) [reply src/components/share-results.tsx]: That's fine, I didn't get the desired layout. Had to do an additional little change, looks good now!

**Pull request merged**